### ISSUE TITLE
Connect to two PVA bot as a skill in Bot Framework Composer

### ISSUE THREAD

**Issue by ErnestoMartins** [microsoft/BotFramework-Composer#9213]
## Describe the bug
Hello I am not sure if it is a bug, but I was required to add two different PVA Bot as a skill for composer. I have added one successfully, the issue is when I try to add the other one I get the error saying "This operation cannot be completed. The skill is already part of the Bot Project" even though the Bot name and ID are completely different. is it possible to do it? is so how can I do it?

## Version
BotComposer version v2.1.

## Browser

- [ ] Electron distribution
- [x ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [x ] Windows
- [ ] Ubuntu

## Expected behaviour

Connect to two different PVA bots as a skill in Bot Framework Composer

## Screenshots

![image](https://user-images.githubusercontent.com/104365881/170713010-5ca03c42-b779-4448-bf7c-e4d4c7258939.png)

**Comment by stevkan** [comment]
@ErnestoMartins, I was able to repro this issue where, after connecting one PVA bot as a skill, if I attempt to connect a second PVA bot as a skill the above error is displayed. The end result is the second PVA bot is not added as a skill.

I'm going to connect with the dev team decide next actions. An update will be posted when we know more.

**Comment by stevkan** [comment]
@ErnestoMartins, I was just comparing the `manifest.json` file found in each .zip file to each other. What I found was that, when unzipping, the files themselves had the same name (not entirely surprising) with one trying to overwrite the other. This got me thinking, so I unzipped the one that was failing, renamed the manifest file to `manifest1.json`, zipped it back up, attempted to add it as a skill, and this worked!

I'm not sure whether the issue is that PVA should be exporting uniquely named manifest files or (more likely) if Composer should be renaming the manifest files as they are imported to avoid collision.

In any event, as a workaround, if you manually rename any additional manifest files (ensuring they still have the `.json` extension, this should mitigate the issue. Please test this and let me know if this works.

**Comment by ErnestoMartins** [comment]
Yes it did work for me as well, thank you very much.

**Comment by LeeParrishMSFT** [comment]
@clearab This is something where we may want to document the workaround, and look at a fix for a future release of Composer to correct this issue.

**Comment by pckyn** [comment]
@ErnestoMartins , @stevkan - When we try to add another skill by renaming **intents1.lu** and **manifest1.json** file , it fails and getting below error..
<img width="949" alt="image" src="https://user-images.githubusercontent.com/126873481/222715867-c7c8fc5b-e1a7-4861-88f9-7ca3889ea0ec.png">

Are we missing anything?

**Comment by manthenavamsi** [comment]
@pckyn : when you rename your intents.lu file to intents1.lu, give that reference in manifest1.json.   At the end of the manifest1.json file, before "Intents" section there is "dispatchModels" in that "dispatchModels" change the "url" key from "intents.lu" to "intents1.lu".  This should fix your problem.

**Comment by pckyn** [comment]
Yes @manthenavamsi .. we tried this approach .. and worked .. thank you..
